www/js/intro.js

```diff
@@ -19,7 +19,27 @@ angular.module('emission.intro', ['emission.splash.startprefs',
 })
 
 .controller('IntroCtrl', function($scope, $state, $ionicSlideBoxDelegate,
-    $ionicPopup, $ionicHistory, ionicToast, $timeout, CommHelper, StartPrefs) {
+    $ionicPopup, $ionicHistory, ionicToast, $timeout, CommHelper, StartPrefs, $translate, $cordovaFile) {
+  
+  $scope.getConsentFile = function () {
+    var lang = $translate.use();
+    $scope.consentFile = "templates/intro/consent.html";
+    if (lang != 'en') {
+      var url = "www/i18n/intro/consent-" + lang + ".html";
+      $cordovaFile.checkFile(cordova.file.applicationDirectory, url).then( function(result){
```

Comment:
Does `cordova.file.applicationDirectory` work on both android and iOS? I generally use `$http.get` to read files from the `www` directory (e.g. `$http.get(`templates/intro/consent.html`) which does work crossplatform, and you would use the http status to determine whether the file existed or not.

Reply:
I tried it in both OS and it worked but I can use `$http.get` if you prefer.

Reply:
It's fine. If I do try to remove the cordova file plugin later, I can change this to `$http.get`